rllib/agents/pg/pg.py

```diff
@@ -1,9 +1,32 @@
+"""
+Implementation of a vanilla policy gradients algorithm. Based on:
+
+[1] - Policy Gradient Methods for Reinforcement Learning with Function
```

Comment:
Please remove this section in favor of a link to the docs.

Reply:
done